src/java/fr/paris/lutece/util/sql/DAOUtil.java

```diff
@@ -256,7 +256,7 @@ public DAOUtil( String strSQL, Integer autoGeneratedKeys, Plugin plugin, boolean
                         }
                         if ( _logger.isDebugEnabled( ) )
```

Comment:
It is unnecessary to check if logging is enabled because it is already managed with the debug level.